src/events/components/DetailView/AttendanceEvent.tsx

```diff
@@ -55,6 +56,9 @@ const AttendanceEvent = ({ event }: IAttendanceEventProps) => {
   const registrationStart = DateTime.fromISO(event.registration_start);
   const registrationEnd = DateTime.fromISO(event.registration_end);
   const cancellationDeadline = DateTime.fromISO(event.unattend_deadline);
+  const sitekey: string = process.env.RECAPTCHA_PUBLIC_KEY || '';
```

Comment:
This should be defined in the `/src/common/constants` folder like everything else we get from env